### PR TITLE
Ian/just class

### PR DESCRIPTION
**Pull request by ianmacartney**
let me know if you want me to back out the relative import of convex from #17 

----

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: template-component/src/client/index.ts [hunk 1]
```diff
@@ -31,26 +31,18 @@ export class Client<Shards extends Record<string, number>> {
   ) {
     return ctx.runQuery(this.component.public.count, { name });
   }
-}
-
-// Another way of exporting functionality
-export function defineCounter(
```
Comment (ianmacartney) [inline template-component/src/client/index.ts:37]: this is the core of this PR - not showing the non-class way for now

**Comment by thomasballinger** [comment]
Is there a type you're hitting that needs it? Fine for now, I've got a doc with some options

**Review by thomasballinger — APPROVED** [review]
sry didn't approve before, lgtm

**Comment by ianmacartney** [comment]
Ok yeah we can back out the relative import once we get things like the
Migrations repro working without it, or can make it the default now to
catch regressions earlier.

**Pull request merged**